package.json

```diff
@@ -5,7 +5,7 @@
   "main": "lib/index.js",
   "scripts": {
     "lint": "grunt eslint",
-    "test": "grunt",
+    "test": "grunt && mocha test/lib",
```

Comment:
`mocha test/lib` is already launched by travis, by doing that you launched it twice